flux-manifests/prometheus-meta-operator.yaml

```diff
@@ -2,7 +2,7 @@ api_version: generators.giantswarm.io/v1
 app_catalog: control-plane-catalog
 app_destination_namespace: monitoring
 app_name: prometheus-meta-operator
-app_version: 4.23.0
+app_version: 4.23.0-8bf8b2b1a8fb29fb8643612f77dccc2cb7683343
```

Comment:
let's accept this for now until
https://github.com/giantswarm/prometheus-meta-operator/pull/1184
is merged